composeApp/src/commonMain/kotlin/sp/bvantur/tasky/agenda/domain/model/AgendaType.kt

```diff
@@ -0,0 +1,17 @@
+package sp.bvantur.tasky.agenda.domain.model
+
+import androidx.compose.ui.graphics.Color
+import org.jetbrains.compose.resources.StringResource
+import sp.bvantur.tasky.core.ui.theme.Green
+import sp.bvantur.tasky.core.ui.theme.Light2
+import sp.bvantur.tasky.core.ui.theme.LightGreen
+import tasky.composeapp.generated.resources.Res
+import tasky.composeapp.generated.resources.event
+import tasky.composeapp.generated.resources.reminder
+import tasky.composeapp.generated.resources.task
+
+enum class AgendaType(val text: StringResource, val color: Color) {
+    EVENT(Res.string.event, LightGreen),
+    TASK(Res.string.task, Green),
+    REMINDER(Res.string.reminder, Light2)
```

Comment:
You're coupling visual stuff to your domain layer here which is not really what should be in there. Both UI texts and colors are something only useful in the presentation layer, so if you want to strictly stick to this layering I would just keep this a blank enum and map it to string resources and colors in the UI.
This will also become necessary in case you decide to make this domain module a real Gradle module, since then you'd need to couple it to Compose resources in order to use these here